tpch/queries/sf=1/q1.sql

```diff
@@ -0,0 +1,23 @@
+-- SQLBench-H query 1 derived from TPC-H query 1 under the terms of the TPC Fair Use Policy.
+-- TPC-H queries are Copyright 1993-2022 Transaction Processing Performance Council.
```

Comment:
Hmm, does different scale factor have different queries? I thought that the query is same?

Reply:
It looks like I misunderstood this. I had generated by specifiying different scale factors and saw different literal values in the generated queries but I think this the generator uses random values for some literals. I have removed the copies.

Reply:
Actually, the TPC-DS qgen does generate specific to scale. See the `SCALE` parameter here. I think we can worry about this later though.

```
qgen2 Population Generator (Version 3.2.0)
Copyright Transaction Processing Performance Council (TPC) 2001 - 2021


USAGE: qgen2 [options]

Note: When defined in a parameter file (using -p), parmeters should
use the form below. Each option can also be set from the command
line, using a form of '-param [optional argument]'
Unique anchored substrings of options are also recognized, and 
case is ignored, so '-sc' is equivalent to '-SCALE'

General Options
===============
FILE =  <s>              -- read parameters from file <s>
VERBOSE =  [Y|N]         -- enable verbose output
HELP =  [Y|N]            -- display this message
OUTPUT_DIR =  <s>        -- write query streams into directory <s>
QUIET =  [Y|N]           -- suppress all output (for scripting)
STREAMS =  <n>           -- generate <n> query streams/versions
INPUT =  <s>             -- read template names from <s>
SCALE =  <n>             -- assume a database of <n> GB
LOG =  <s>               -- write parameter log to <s>
QUALIFY =  [Y|N]         -- generate qualification queries in ascending order
```

Reply:
Interesting. I used to use TPC-DS or TPC-H benchmarks like https://github.com/databricks/spark-sql-perf, and what we have in Comet or Spark. I remember we only specify scale during generating the data, and the queries are the same (it is not generated).

Maybe it is specified to this tool you use. It is okay for me if the query generator considers scale.